shared/styles/shared.tsx

```diff
@@ -79,20 +79,32 @@ const unifyStyles = (s: any) => ({
     : {}),
 })
 
-export const platformStyles = (options: {
-  common?: _StylesCrossPlatform
-  isIOS?: _StylesMobile
-  isAndroid?: _StylesMobile
-  isMobile?: _StylesMobile
-  isElectron?: _StylesDesktop
-  // TODO any for now, but we need the types to be handled differently
-}): any => ({
-  ...(options.common ? unifyStyles(options.common) : {}),
-  ...(isMobile && options.isMobile ? options.isMobile : {}),
-  ...(isIOS && options.isIOS ? options.isIOS : {}),
-  ...(isAndroid && options.isAndroid ? options.isAndroid : {}),
-  ...(isElectron && options.isElectron ? unifyStyles(options.isElectron) : {}),
-})
+type AsStylesCrossPlatform<T> = Readonly<
```

Comment:
What's this type for?

Reply:
this is what platformStyles returns below. It casts the keys as the values of cross platform so things that consume stylescrossplatform can accept the styles. else if you use this helper you can't use the styles anymore. later we can ditch styles cross platform and have the actual styles